authorization-common-api/src/main/java/com/sequenceiq/authorization/resource/AuthorizationResourceAction.java

```diff
@@ -81,6 +81,7 @@
     POWERUSER_ONLY("cloudbreak/allowPowerUserOnly", null),
     LIST_ASSIGNED_ROLES("iam/listAssignedResourceRoles", null),
     STRUCTURED_EVENTS_READ("structured_events/read", AuthorizationResourceType.STRUCTURED_EVENT),
+    COLLECT_USED_IMAGES("cloudbreakadmin/collectUsedImages", null),
```

Comment:
do we plan to introduce a specific admin role for this? I am asking this only for curiosity

Reply:
the plan is to create a machine user with only this permission so we can call these endpoints from a Jenkins job, using the internal cdp cli

Reply:
if this meant to used with internal cdp cli only, then I think cdp cli should use internal actor and we should allow these endpoints only for internal actor, then we do not need to add a new right to UMS, what do you think about this?

Reply:
It's not that simple, we've had a discussion with UMS. 
This is the way Admin API is meant to be used. You can't authenticate as an internalActor though the coreadmin api.

Reply:
Interesting. I just realised that this API can be called by any PowerUser, which was not the intention I think.
At least our framework will just ask UMS about this right and PowerUser has it all.

Is there any separate logic that distinguish internal rights?

Reply:
I don't think so. I missed your pervious comment yesterday, so if internalactor usage is not possible, then makes sense to add a new right.

Reply:
This right was removed (it will be checked in the api service), and marked the endpoints InternalOnly